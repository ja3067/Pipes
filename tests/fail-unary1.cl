x = 3
b = 3

if x == 3:
    b = "a"

c = -b
