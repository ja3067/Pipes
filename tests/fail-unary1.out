RuntimeError: unsupported operand type for unary -
